Formula/musikcube.rb

```diff
@@ -0,0 +1,42 @@
+class Musikcube < Formula
+  desc "Terminal-based audio engine, library, player and server"
+  homepage "https://musikcube.com"
+  url "https://github.com/clangen/musikcube/archive/0.99.4.tar.gz"
+  sha256 "dd8e74c8c8460250c0da325941db2df0f7dcca77f2c68a0295ab21e81f8dd8a4"
+  license "BSD-3-Clause"
+  head "https://github.com/clangen/musikcube.git", branch: "master"
+
+  depends_on "ccache" => :build
```

Comment:
Is this a soft or hard dependency? Pretty sure this cache won't persist.

Reply:
Yeah, you're right, this can be removed. Fixed. https://github.com/Homebrew/homebrew-core/pull/118938/commits/8d065567c7dd5eff57977fdd0f131fea43a54cec